.gitignore

```diff
@@ -1,5 +1,6 @@
 /config/*
 !/config/formats.ts
+!/config/custom-formats.ts
```

Comment:
```suggestion
```

In theory this should no longer be needed.

Reply:
Sorry for not responding to this. I had gone to bed at this point. Thank you for just going ahead and making the change.